src/ds/ds6/hw_monitor_extended_buffers.h

```diff
@@ -20,5 +28,24 @@ namespace librealsense
     private:
         int get_msg_length(command cmd) const;
         int get_number_of_chunks(int msg_length) const;
+        hwm_buffer_type get_buffer_type(command cmd) const;
+        std::vector<uint8_t> send_big_buffer_to_receive(command cmd, hwmon_response* p_response, bool locked_transfer) const;
+        std::vector<uint8_t> send_big_buffer_to_send(command cmd, hwmon_response* p_response, bool locked_transfer) const;
+
+        // The following method prepares the param4 of the command for extended buffers
+        // The aim of this param is to send the chunk number out of max of expected chunks for the current command
+        // It results to a 32 bit integer, which is built from two 16 bits ints:
+        // Chunk Id / Max Chunk id
+        // uint16(LSB / Little Endian) - current chunk id(0�N - 1)
```

Comment:
![image](https://user-images.githubusercontent.com/64067618/214683253-18f8d0e8-b4f2-4501-a1c1-58c29847bd1f.png)
 Don't know if its just on github but that's a weird symbol.

Reply:
corrected